### PR TITLE
Modify string length/size entries

### PR DESCRIPTION
**Pull request by Dusch4593**
### Description

For SEO keyword efforts

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/string-functions/terms/strlen/strlen.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+Title: 'strlen()'
+Description: 'Returns the number of byte in a given string.'
```
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/strlen/strlen.md:3]:
```suggestion
Description: 'Returns the number of bytes in a given string.'
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Dusch4593, I did a first review and only had a couple of comments.

**Comment by Dusch4593** [comment]
@SSwiniarski Implemented your feedback 😄

**Review by SSwiniarski — APPROVED** [review]
@Dusch4593, Looks good for review two.

File: content/c/concepts/strings/terms/strlen/strlen.md [hunk 2]
```diff
@@ -8,22 +8,22 @@ Tags:
...
-  - 'Arrays'
   - 'Functions'
+  - 'Methods'
 CatalogContent:
   - 'learn-c'
   - 'paths/computer-science'
 ---
 
-The `strlen()` string function returns the length of the string without including terminating character `'\n'`.
+The **`strlen()`** string function returns the length of the string without including terminating character `'\n'`.
```
Comment (KTom101) [inline content/c/concepts/strings/terms/strlen/strlen.md:18]:
```suggestion
The **`strlen()`** string function returns the length of the string without including the terminating character, `'\n'`.
```

File: content/javascript/concepts/strings/strings.md [hunk 3]
```diff
@@ -13,30 +13,13 @@ CatalogContent:
...
-console.log(message.length);
-// Output: 9
-
-console.log('howdy'.length);
-// Output: 5
-```
+## Concatenation
 
-## String Concatenation
+In JavaScript, multiple strings can be concatenated, or joined, together using the `+` [operator](https://www.codecademy.com/resources/docs/javascript/operators).
```
Comment (KTom101) [inline content/javascript/concepts/strings/strings.md:20]:
```suggestion
In JavaScript, multiple strings can be concatenated or joined together using the `+` [operator](https://www.codecademy.com/resources/docs/javascript/operators).
```

File: content/javascript/concepts/strings/strings.md [hunk 4]
```diff
@@ -13,30 +13,13 @@ CatalogContent:
...
 
-In JavaScript, multiple strings can be concatenated together using the `+` operator.
-
-In the example, multiple strings and variables containing string values have been concatenated. After execution of the code block, the displayText variable will contain the concatenated string.
+In the example below, multiple strings and variables containing string values have been concatenated. After code block is run, the `displayText` variable will contain the concatenated string:
```
Comment (KTom101) [inline content/javascript/concepts/strings/strings.md:22]:
```suggestion
In the example below, multiple strings and variables containing string values have been concatenated. After the code block is run, the `displayText` variable will contain the concatenated string:
```

File: content/javascript/concepts/strings/terms/length/length.md [hunk 5]
```diff
@@ -13,17 +14,19 @@ CatalogContent:
...
+The **`.length`** property returns the number of characters in a string.
 
 ## Syntax
 
-```js
+```pseudo
 string.length;
 ```
 
+The `string` can either be a literal sequence of characters surrounded in quotes or a variable that refers to such a sequence.
```
Comment (KTom101) [inline content/javascript/concepts/strings/terms/length/length.md:25]:
```suggestion
The `string` can either be a literal sequence of characters surrounded by quotes or a variable that refers to such a sequence.
```

File: content/php/concepts/string-functions/terms/strlen/strlen.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+The **`strlen()`** method returns the number of bytes in a given string.
+
+## Syntax
+
+```pseudo
+strlen(string);
+```
+
+The `string` can either be a literal sequence of characters surrounded in quotes or a variable that refers to such a sequence.
```
Comment (KTom101) [inline content/php/concepts/string-functions/terms/strlen/strlen.md:24]:
```suggestion
The `string` can either be a literal sequence of characters surrounded by quotes or a variable that refers to such a sequence.
```

File: content/php/concepts/string-functions/terms/strlen/strlen.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+## Codebyte Example
+
+In the following example showcases the two ways of using the `strlen()` method:
```
Comment (KTom101) [inline content/php/concepts/string-functions/terms/strlen/strlen.md:28]:
```suggestion
The following example showcases the two ways of using the `strlen()` method:
```

File: content/ruby/concepts/strings/terms/length/length.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+
+The **`.length`** property returns the number of characters in a string.
+
+## Syntax
+
+```pseudo
+string.length
+```
+
+The `string` can either be a literal sequence of characters surrounded in quotes or a variable that refers to such a sequence.
```
Comment (KTom101) [inline content/ruby/concepts/strings/terms/length/length.md:25]:
```suggestion
The `string` can either be a literal sequence of characters surrounded by quotes or a variable that refers to such a sequence.
```

File: content/cpp/concepts/strings/terms/size/size.md [hunk 9]
```diff
@@ -6,20 +6,24 @@ Subjects:
...
+  - 'Characters'
+  - 'Functions'
   - 'Methods'
 CatalogContent:
   - 'learn-c-plus-plus'
   - 'paths/computer-science'
 ---
 
-The `.size()` method returns the size of a given string in terms of bytes. It functions just like the [`.length()`](https://www.codecademy.com/resources/docs/python/strings/length) method, but `.size()` is there to follow the consistency of other standard library containers (e.g., maps and [vectors](https://www.codecademy.com/resources/docs/cpp/vectors)).
+The **`.size()`** method returns the size of a given string in terms of bytes. It functions just like the [`.length()`](https://www.codecademy.com/resources/docs/python/strings/length) method, but `.size()` is there to follow the consistency of other standard library containers (e.g., maps and [vectors](https://www.codecademy.com/resources/docs/cpp/vectors)).
```
Comment (KTom101) [inline content/cpp/concepts/strings/terms/size/size.md:17]:
```suggestion
The **`.size()`** method returns the size of a given string in terms of bytes. It functions just like the [`.length()`](https://www.codecademy.com/resources/docs/python/strings/length) method, but `.size()` is there to follow the consistency of other standard library containers (e.g., [maps](https://www.codecademy.com/resources/docs/cpp/maps) and [vectors](https://www.codecademy.com/resources/docs/cpp/vectors)).
```

File: content/cpp/concepts/strings/terms/size/size.md [hunk 10]
```diff
@@ -6,20 +6,24 @@ Subjects:
...
 
 ## Syntax
 
 ```pseudo
-stringName.size();
+string.size();
 ```
 
+The `string` must be defined with `std::string` prior to using to the `.size()` method.
```
Comment (KTom101) [inline content/cpp/concepts/strings/terms/size/size.md:25]:
```suggestion
The `string` must be defined with `std::string` prior to using with the `.size()` method.
```

**Review by KTom101 — CHANGES_REQUESTED** [review]
@Dusch4593 Added a few changes!

**Comment by Dusch4593** [comment]
@KTom101 Finished implementing your suggested changes 😄

File: content/cpp/concepts/strings/terms/length/length.md [hunk 11]
```diff
@@ -6,20 +6,23 @@ Subjects:
...
+The **`.length()`** method returns the length of the given string in terms of bytes. It functions exactly like the [.size()](https://www.codecademy.com/resources/docs/python/strings/size) method but follows a more intuitive terminology (i.e., "the length of a string" rather than "the size of a string"). However, both methods yield the same result.
 
 ## Syntax
 
 ```pseudo
-stringName.length();
+string.length();
 ```
 
+The `string` must be defined with `std::string` prior to using to the `.length()` method.
```
Comment (KTom101) [inline content/cpp/concepts/strings/terms/length/length.md:24]:
```suggestion
The `string` must be defined with `std::string` prior to using with the `.length()` method.
```

**Review by KTom101 — APPROVED** [review]
@Dusch4593 just one more thing but otherwise looks great!

**Pull request merged**